### PR TITLE
Update veil_falls.logic

### PR DESCRIPTION
**Pull request by CandyCrystal**

### CODE REVIEW

**Review by 21aslade — CHANGES_REQUESTED** [review]

File: veil_falls.logic [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+      door SouthWest
+    Bottom <-> BottomWater: item Flippers
+    Bottom <-> BottomEast: (item Cape | Item Flippers)
+    node BottomEast:
+      chest Rupee1
+      chest Rupee2
+      chest Rupee3
+      enemy LikeLike20
+      exit BottomEastDigCave
+    room BottomEastDigCave:
```
Comment (21aslade) [inline veil_falls.logic:20]:
```suggestion
    node BottomEastDigCave:
```

File: veil_falls.logic [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+      entrance BottomEastDigCave
+    BottomEast -> BottomEastDigCave: item MoleMitts
```
Comment (21aslade) [inline veil_falls.logic:22]:
```suggestion
    BottomEast -> BottomEastDigCave: item MoleMitts
    BottomEastDigCave -> BottomEast
```

File: veil_falls.logic [hunk 3]
```diff
@@ -1,2 +1,77 @@
...
+    MiddleMain <-> MiddleHP: (item Cape | item Flippers)
+    node MiddleDigCaveFusion:
+      door MiddleDigDoor
+    MiddleHP <-> MiddleDigCaveFusion: flag VeilFallsDigFusion
+    node MiddleChestLedge:
+      chest MiddleLedgeChest
+      door MiddleLedgeCave
+    node Crest:
+      entrance Crest
+      entrance TopCaveMiddle
```
Comment (21aslade) [inline veil_falls.logic:44]:
```suggestion
      door TopCaveMiddle
```

**Review by 21aslade — CHANGES_REQUESTED** [review]
Didn't spot these first run through

File: veil_falls.logic (hunk 2 above)
Comment (CandyCrystal) [reply veil_falls.logic:22]: Is that neccesary when the exit part of the door is in BottomEast?

Comment (21aslade) [reply veil_falls.logic:22]: It's not, you're right. My bad

**Review by 21aslade — APPROVED** [review]

**Pull request merged**